### PR TITLE
Embed fixes

### PR DESCRIPTION
**Pull request by otter-computer**
- Fixes some detection logic in the nickname change event that was accidentally left enabled, it should work properly now.
- Changes the URL that deleted messages event uses to render attachments. This should mean we can _likely always_ see deleted selfies uploaded directly

### CODE REVIEW

File: commands/joined.js [hunk 1]
```diff
@@ -1,33 +1,42 @@
...
-      const joined = moment(message.guild.member(user).joinedAt);
-      message.channel.sendMessage(user + ' joined ' + joined.fromNow() +
-          ' (' + joined.format(format) + ')');
+      let embed = new Discord.RichEmbed();
+
+      embed.setColor(0x3398DB);
+      embed.setTitle(member.displayName + ' joined ' + moment(joined).fromNow());
+
+      if (user.id === REYNBOW.id) {
+        embed.setTitle(member.displayName.toUpperCase() +
```
Comment (ThomasShaped) [inline commands/joined.js]: Why did I do that

File: events/memberUpdated.js [hunk 2]
```diff
@@ -1,36 +1,51 @@
...
+  // Create the embed
+  let embed = new Discord.RichEmbed();
+  embed.setColor(0xF1C40E);
+  embed.addField('User', newMember);
 
-  if (!oldMember.nickname) {
-    oldMemberName = oldMember.user.username;
-  } else {
-    oldMemberName = oldMember.nickname;
+  let embedDate = new Date(Date.now());
```
Comment (jmariondev) [inline events/memberUpdated.js]: Why not remove this line and just add it to the below line?

File: events/memberUpdated.js [hunk 3]
```diff
@@ -1,36 +1,51 @@
...
+  embed.setTimestamp(embedDate.toISOString());
```
Comment (jmariondev) [inline events/memberUpdated.js]: Is there a reason this is `toISOString()` here, but only supplying a `Date` object above?

File: events/memberUpdated.js [hunk 4]
```diff
@@ -1,36 +1,51 @@
...
+
+  // User adds a nicknameUser changed account username
```
Comment (jmariondev) [inline events/memberUpdated.js]: o.0

File: events/memberUpdated.js [hunk 5]
```diff
@@ -39,18 +54,20 @@ function memberRestricted(member) {
...
-        () => {},
-        reason => {
-          // TODO Rejection handler
-          console.error(reason);
-        }
+    () => { },
+    reason => {
+      // TODO Rejection handler
+      console.error(reason);
+    }
```
Comment (jmariondev) [inline events/memberUpdated.js]: Is this indentation change a deliberate style choice?

**Review by jmariondev — CHANGES_REQUESTED** [review]
Alright, I added my comments 😄

File: events/memberUpdated.js (hunk 3 above)
Comment (otter-computer) [reply events/memberUpdated.js]: No! 😄

I mean the `toISOString` is required as discord fucking *dies* if you give it _anything else_ in that embed slot, but I don't know why I didn't just `Date.now().toISOString()` and skipped the date variable.

File: events/memberUpdated.js (hunk 5 above)
Comment (otter-computer) [reply events/memberUpdated.js]: ESlint gave me indentation squiglies about the formatting here. 

EDIT: wait, it complained, but on second opening it's complaining again. **I CAN'T WIN.** (I will revert)

File: events/memberUpdated.js (hunk 4 above)
Comment (otter-computer) [reply events/memberUpdated.js]: \>_>;

**Review by otter-computer — COMMENTED** [review]
Updated your requested changes @LuaMilkshake 😁😁😁

**Pull request merged**